tst/regression/regression_test.py

```diff
@@ -216,6 +246,17 @@ def gold_comparison(
     compression_factor=1,
     tolerance=1.0e-5,
 ):
+    problem = read_input_value("phoebus", "problem", input_file)
```

Comment:
It was difficult to understand what tests weren't working from the CI without this.